velox/exec/MergeSource.h

```diff
@@ -74,6 +74,7 @@ class MergeJoinSource {
   struct State {
     bool atEnd = false;
     RowVectorPtr data;
+    std::queue<RowVectorPtr> dataQueue;
```

Comment:
in which case, we have more than one output?

Reply:
When executing a 1GB TPC-H Query 21 using multiple executors and multiple cores, the issue will arise.